app/views/entities/new.html.erb

```diff
@@ -0,0 +1,53 @@
+
+
+
+<%= form_with model: @entity, url: group_entities_path, method: :post, html: { class: "form-logged-in" } do |form| %>
+
+<%= link_to groups_path do %>
+  <i class="fa-solid fa-circle-chevron-left back-icon "></i>
```

Comment:
- [ ] Based on the requirements of this project, when a user clicks on the `back` button, he/she should navigate back to the transaction page for that particular category and not the home page.